### PR TITLE
fix: address linter issues and enable github action with linter

### PR DESCRIPTION
**Pull request by dshulyak**
Changes are straightforward and only addressing reported issues.
I disabled `interfacer` linter, as it doesn't always report relevant things,
also disabled rule G404: weak randomness linter, it recommends using crypto.Rand
even when it is unnecessary.

closes: https://github.com/tendermint/spn/issues/119

### CODE REVIEW

**Comment by dshulyak** [comment]
@fadeev @lubtd who can add me to tendermint org or at least enable CI for this PR?

**Comment by fadeev** [comment]
@dshulyak I'll be adding you to both repos now!

**Comment by ilgooz** [comment]
@dshulyak for the time being, I triggered the checks for this PR manually. (your following commits should also work since you're now authorized with this one)

File: internal/utils/parse_test.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	}{
+		{
+			description: "regular",
+			input:       "10",
+			result:      10,
+		},
+		{
+			description: "overflow",
+			input:       strconv.Itoa(math.MaxInt64),
+			err:         "strconv.ParseInt: parsing \"%s\": value out of range",
```
Comment (ilgooz) [inline internal/utils/parse_test.go:27]: Instad of comparing strings, can we unwrap the received error and compare with defined error values.
e.g.: https://golang.org/pkg/strconv/#pkg-variables

**Review by ilgooz — CHANGES_REQUESTED** [review]
Congrats on the first PR!!!! 🎊 🍪

File: internal/utils/parse.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package utils
```
[the remark this answers is not in the document]
Comment (dshulyak) [reply internal/utils/parse.go:1]: i don't think that it is meant to be reusable outside of spn

**Comment by dshulyak** [comment]
@ilgooz do you want me to fix more things?

**Review by ilgooz — APPROVED** [review]
It is very good, thank's for changes.

**Review by lubtd — APPROVED** [review]
Thanks, it's a great PR

**Pull request merged**